site/_docs/history.md

```diff
@@ -49,6 +49,12 @@ other software versions as specified in gradle.properties.
 #### Breaking Changes
 {: #breaking-1-42-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-7301">CALCITE-7301</a>]
```

Comment:
It is a breaking change, but it should not affect anyone. All `SqlNode` classes inside `org.apache.calcite.sql.ddl` expose their operands through public fields, and none of these nodes implement `setOperand`. Because of this, the change won’t impact existing code.

For `SqlUnpivot`, I added the SqlLiteral at the end of the operand list.